src/cmakeProject.ts

```diff
@@ -1746,7 +1748,9 @@ export class CMakeProject {
                 }
             }
         } else if (!this.configurePreset) {
-            throw new Error(localize('cannot.configure.no.config.preset', 'Cannot configure: No configure preset is active for this CMake project'));
+            await vscode.window.showErrorMessage(localize('cannot.configure.no.config.preset', 'Cannot configure: No configure preset is active for this CMake project'));
```

Comment:
I thought we were going to remove the error message? Applies for above as well

Reply:
I thought we were just changing it from the ugly throw new Error to a show error message? Should it be an info message instead? I think it should still be an error message.

Reply:
Ah, you're exactly right, I got confused from the other locations. Thanks.